moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -215,6 +211,7 @@ const ComponentForm = ({
             disabled={isSignalComponent && isEditingExistingComponent}
             autoFocus
             helperText="Required"
+            required={true}
```

Comment:
for consistency, this makes the asterisk (`*`) appear next to the field name on the form.